k8_cortx_cloud/destroy-cortx-cloud.sh

```diff
@@ -400,55 +378,49 @@ function delete3rdPartyPVs()
         fi
         kubectl delete pv "${persistent_volume}"
     done
-
-    if [[ ${namespace} != 'default' ]]; then
-        persistent_volumes=$(kubectl get pv --namespace="${namespace}" | grep -E "${pvc_consul_filter}|${pvc_kafka_filter}|${pvc_zookeeper_filter}|cortx|3rd-party" | cut -f1 -d " ")
-        [[ -n ${persistent_volumes} ]] && echo "${persistent_volumes}"
-        for persistent_volume in ${persistent_volumes}
-        do
-            printf "Removing %s\n" "${persistent_volume}"
-            if [[ "${force_delete}" == "--force" || "${force_delete}" == "-f" ]]; then
-                kubectl patch pv "${persistent_volume}" -p '{"metadata":{"finalizers":null}}'
-            fi
-            kubectl delete pv "${persistent_volume}"
-        done
-    fi
 }
 
 function deleteStorageProvisioner()
 {
+    # WWW:  TODO: This is installed the first time cortx is installed.  If there are multiple
+    # cortx deployments, I don't want this to be removed when the first cortx installation
+    # is removed!
     rancher_prov_path="$(pwd)/cortx-cloud-3rd-party-pkg/auto-gen-rancher-provisioner"
     rancher_prov_file="${rancher_prov_path}/local-path-storage.yaml"
     [[ -f ${rancher_prov_file} ]] && kubectl delete -f "${rancher_prov_file}"
     rm -rf "${rancher_prov_path}"
 }
 
-function helmChartCleanup()
-{
-    printf "########################################################\n"
-    printf "# Uninstalling leftover Helm Charts                    #\n"
-    printf "########################################################\n"
-    print_header=true
-    local charts
-    charts="$(helm list | { grep '^consul\|^cortx\|^kafka\|^openldap\|^zookeeper' || true; })"
-    if [[ -n ${charts} ]]; then
-        while IFS= read -r line; do
-            IFS=" " read -r -a my_array <<< "${line}"
-            if [[ ${print_header} = true ]]; then
-                printf "Helm chart cleanup:\n"
-                print_header=false
-            fi
-            uninstallHelmChart "${my_array[0]}" default
-        done <<< "${charts}"
-    fi
-}
+#function helmChartCleanup()
```

Comment:
I commented out this entire function.  It's not obvious to me that it is needed, though I am not 100% sure yet.  I need to review more closely and make a final determination.

Reply:
This has since been removed.